README.md

```diff
@@ -101,6 +101,200 @@ any zone.
 
 You can define an arbitrary number of drag items.
 
+Analytics Events
```

Comment:
We could just verbatim copy the design page. I put the actual output because it would be reflective of what all the person handling analytics would be able to get, which seemed most important to me.